natural-language-understanding/src/main/java/com/ibm/watson/developer_cloud/natural_language_understanding/v1/model/CategoriesResult.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.natural_language_understanding.v1.model;
+
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * The hierarchical 5-level taxonomy the content is categorized into.
+ */
+public class CategoriesResult extends GenericModel {
+
+  /** The path to the category through the taxonomy hierarchy. */
+  private String label;
+  /** Confidence score for the category classification. Higher values indicate greater confidence. */
+  private Double score;
+
+  /**
+   * Gets the label.
+   *
+   * @return the label
+   */
+  public String getLabel() {
```

Comment:
Getter name should be "label" rather than "getLabel"; "score" rather than "getScore".